config/crd/bases/authorino-operator.kuadrant.3scale.net_authorinoes.yaml

```diff
@@ -0,0 +1,133 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.6.1
+  creationTimestamp: null
+  name: authorinoes.authorino-operator.kuadrant.3scale.net
+spec:
+  group: authorino-operator.kuadrant.3scale.net
+  names:
+    kind: Authorino
+    listKind: AuthorinoList
+    plural: authorinoes
```

Comment:
I wonder whether a more natural spelling wouldn't be "authorinos".

(BTW I wish I could pluralize it as "authorini", but people would just kill me 🙂)

Reply:
I think it's the default way that the `operator-sdk` pluralize the name of the objects, I am happy to change to  `authorinos` if it's not going against any standards.